apis/v1alpha3/backendtlspolicy_types.go

```diff
@@ -124,15 +130,15 @@ type BackendTLSPolicyConfig struct {
 	//    served by the matching backend.
 	//
 	// Support: Core
-	Hostname v1beta1.PreciseHostname `json:"hostname"`
+	Hostname v1.PreciseHostname `json:"hostname"`
 }
 
-// WellKnownCACertType is the type of CA certificate that will be used when
+// WellKnownCACertificatesType is the type of CA certificate that will be used when
 // the TLS.caCertRefs is unspecified.
```

Comment:
nit, it seems this is the only field didn't get renamed 
```suggestion
// the TLS.caCertificateRefs is unspecified.
```

Reply:
Good catch, thanks! https://github.com/kubernetes-sigs/gateway-api/pull/3018